data/human/human missions.txt

```diff
@@ -3996,3 +3996,132 @@ mission "Fumbling Mugger"
+			choice
+				`	"I can help you. We'll make haste to reach <planet> in time."`
+					goto help
+				`	"With that much money, why not buy your own ship?"`
+					goto question
+				`	"I don't have time to take on a passenger right now."`
+			`	She nods, and moves on to another captain to beseech her help instead.`
+				decline
+			label question
+			`	"I wouldn't know how to fly it, and I don't have time to learn. Please, can you help or not?"`
+			choice
+				`	"I can help you. We'll make haste to reach <planet> in time."`
+					goto help
+				`	"I don't have time to take on a passenger right now."`
+			`	She nods, and moves on to another captain to beseech her help instead.`
+				decline
+			label help
+			`	Her eyes widen, and she opens her mouth twice before words come out. "Thank you! This means the world to me. My name is Riley, by the way. I don't have much luggage; I'll be ready to leave as soon as you are."`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Riley hasn't entered this system yet. Better depart and wait for it.`
+	on complete
+		payment 13000
+		conversation
+			`In the days since Riley came aboard, she has relaxed and cleaned up. She looks nearly unrecognizable from the tired, desperate woman you first saw.`
+			`	On the way, she told you stories of her past, mostly of time shared with her father. As a young teen, they spent summers riding the latest horse mechs through the forest, looking for beehives to steal honey from. At age 19, she broke her ankle hoverboarding a restricted mountain peak. Her father rescued her, avoiding the police to hike up with a medical kit. She talks about his wild birthday gifts to her each year, always a different piece of clothing with embedded AI that was just shy of what was allowed by law. As you watch, she shows how her shirt warms and constricts if it detects stress, mimicking a hug.`
+			`	She didn't mention it, but it's clear that she was concerned that she'll arrive too late to say goodbye.`
+			`	As you neared the <system>, Riley told you about the last time she saw her father. He had become focused on his work - research into safe AI - and they hadn't seen each other for years. She had become a well-known DJ back on <source> and had a break in her touring schedule, so she made contact and they met halfway. They spent most of a week together, reliving old times.`
+			`	He'd cried when it was time to leave, and she had, too, but life went on, and it was years before she heard from him again. His message nearly broke her. "Ri, please come as soon as you can. I may not have long, and wish to hold your hand one more time. All love, Dad."`
+			`	Riley directs you to a pad near her old home, and you land the shuttle. She starts toward the door, then pauses and turns around with a self-conscious, but oddly sad, chuckle.`
```

Comment:
iirc many missions(FW when you got attacked by pirates when landed, Momon's Sub, Dreadnought reveal) imply we land with the entire ship not a shuttle, so I think it shouldn't be this specific with the shuttle part.